components/pages/privacy/consent/requests/generate-consent-request/index.jsx

```diff
@@ -48,7 +50,19 @@ export default function GenerateConsentRequest() {
       requestorInboxUrl: profile.inbox,
       options,
     });
-    setConsentId(vc.id);
+    if (vc) {
+      redirectToConsentManagementUi(
+        vc,
+        `https://localhost:3000//privacy/?signedVcUrl=${encodeURIComponent(
+          vc.id
+        )}`,
+        {
+          fallbackConsentManagementUi: `https://localhost:3000/privacy/consent/requests/${encodeURIComponent(
```

Comment:
This is redirecting me to actual localhost:3000 so I can't complete the test :(

Reply:
ugh sorry - very silly mistake. I fixed it now, you should be able to see it now.